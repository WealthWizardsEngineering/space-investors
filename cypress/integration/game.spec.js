describe("Space investors", () => {
  beforeEach(() => cy.visit("http://127.0.0.1:8080"));

  it("should load the game", () => {
    cy.get("canvas")
      .should("be.visible")
      .and("have.css", "width", "800px")
      .and("have.css", "height", "600px");
  });
  it("should match the screenshot", () => {
    cy.wait(1000);
    cy.get("canvas").toMatchImageSnapshot();
  });

  it("should allow player to move both ways", () => {
    cy.wait(500);
    cy.get("canvas").trigger("keydown", {
      keyCode: 39,
    });
    cy.wait(500);
    cy.get("canvas").trigger("keydown", {
      keyCode: 37,
    });
  });
});
