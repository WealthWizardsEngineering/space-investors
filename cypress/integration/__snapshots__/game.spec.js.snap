exports[`Space investors > should match the screenshot #0`] = `
<canvas width="800" height="600"></canvas>
`;
